plutus-core/untyped-plutus-core/test/Evaluation/ApplyBuiltinName.hs

```diff
@@ -58,9 +58,8 @@ instance SpendBudget AppM DefaultFun () (Term Name DefaultUni DefaultFun ()) whe
 test_applyBuiltinFunction :: DefaultFun -> TestTree
 test_applyBuiltinFunction fun =
     testProperty (show fun) . property $ case toBuiltinMeaning fun of
-        BuiltinMeaning sch toF toExF -> do
-            let f = toF ()
-                exF = toExF defaultCostModel
+        BuiltinMeaning sch f toExF -> do
+            let exF = toExF defaultCostModel
             withGenArgsRes sch f $ \args res ->
                 -- The calls to 'unAppM' are just to drive type inference.
                 unAppM (applyTypeSchemed fun sch f exF args) === unAppM (makeKnown res)
```

Comment:
Just below here there's a function called `test_applyStaticBuiltin`.  Maybe that should be`test_applyDefaultBuiltin` or something?